### PR TITLE
ipq806x: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
Splits up DEVICE_TITLE into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT.

### CODE REVIEW

File: target/linux/ipq806x/image/Makefile [hunk 1]
```diff
@@ -144,13 +150,14 @@ define Device/netgear_d7800
...
 	SUPPORTED_DEVICES += d7800
-	DEVICE_TITLE := Netgear Nighthawk X4 D7800
 	DEVICE_PACKAGES := ath10k-firmware-qca99x0-ct
 endef
 TARGET_DEVICES += netgear_d7800
 
 define Device/netgear_r7500
 	$(call Device/DniImage)
+	DEVICE_VENDOR := NETGEAR
+	DEVICE_MODEL := Nighthawk X4 R7500
```
Comment (adrianschmutzler) [inline target/linux/ipq806x/image/Makefile]: If there is a v2 below, this should have VARIANT v1.

File: target/linux/ipq806x/image/Makefile [hunk 2]
```diff
@@ -189,75 +199,82 @@ define Device/netgear_r7800
...
 	DEVICE_PACKAGES := ath10k-firmware-qca99x0-ct
 endef
 TARGET_DEVICES += qcom_ipq8064-ap148
 
 define Device/qcom_ipq8064-ap148-legacy
 	$(call Device/LegacyImage)
 	$(call Device/UbiFit)
+	DEVICE_VENDOR := Qualcomm
+	DEVICE_MODEL := AP148
+	DEVICE_VARIANT := (legacy)
```
Comment (adrianschmutzler) [inline target/linux/ipq806x/image/Makefile]: Brackets or not ...?
Despite, if that's variant "legacy", we need to also provide a variant for the "standard" one.

**Review by adschm — CHANGES_REQUESTED** [review]
Looks okay despite concerning the two comments.

**Review by adschm — APPROVED** [review]

**Comment by adrianschmutzler** [comment]
@mwarning Had a look at my C2600 and it says Ver.1.1.
If you still want, you could also add "v1" to both TP-Link here ...

**Review by adschm — APPROVED** [review]

**Comment by lynxis** [comment]
merged. Thank you!

**Pull request closed without merging**